java/src/client/interpreter/Interpreter.java

```diff
@@ -1,18 +1,364 @@
 package client.interpreter;
 
+import java.lang.reflect.Type;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+
+import com.google.gson.*;
+
+import shared.definitions.HexType;
+import shared.locations.EdgeDirection;
+import shared.locations.EdgeLocation;
+import shared.locations.HexLocation;
+import shared.locations.VertexDirection;
+import shared.locations.VertexLocation;
+import shared.models.*;
 
 
 public class Interpreter {
 
+	GsonBuilder gsonBuilder = new GsonBuilder();
+	Gson gson;
 	
 	
-	public Interpreter() {
+	public Interpreter() {	
+		gson = new Gson();
 		
+		gson = gsonBuilder.create();
```

Comment:
Does the line above even matter then? We create a gson object, then reassign what it is right after